pytest.ini

```diff
@@ -0,0 +1,6 @@
+[pytest]
+env =
+    # System fonts can interfere with screenshots that expect a particular font
+    # to be used
+    PYGFX_DISABLE_SYSTEM_FONTS=1
+addopts = --capture=no -v
```

Comment:
I'm not sure if setting capture=no is an improvement for CI output?

Reply:
ok.
```suggestion
```